app/controllers/v1/bookings_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module V1
+  class BookingsController < ApplicationController
+    before_action :set_meeting_room
+    before_action :set_user
```

Comment:
Wouldn't this be only for the `create` action?

Reply:
Yes as of now there is only create action, In future even Cancel action will come into picture. Till then I will add only: [:create] .